---

software_sync_params:
   software_sync: true
   software_sync_period: 0.055 # seconds, must be greater than 0.055

cameras:
  -
    camera_id: Basler-21135424
    pixel_format: bayer_bggr8
    gain_mode: off
    gain: 70
    exposure_mode: fixed
    exposure: 15000
    auto_white_balance_once: false
    frame_rate: 20
    digital_shift: 0
    bin_x: 0
    bin_y: 0
    roi_x: 0
    roi_y: 0
    roi_width: 1624
    roi_height: 1234
    AcquisitionStartTrigMode: Off
    AcquisitionStartTrigSource: Software
    AcquisitionStartTrigActivation: RisingEdge
    FrameStartTrigMode: On
    FrameStartTrigSource: Software
    FrameStartTrigActivation: RisingEdge
    line_selector: Out1
    line_mode: Output
    line_source: ExposureActive
    user_output_selector: UserOutput1
    info_filename: info_right.ini
  -
    camera_id: Basler-21135423

    pixel_format: bayer_bggr8
    gain_mode: off
    gain: 70
    exposure_mode: fixed
    exposure: 15000
    auto_white_balance_once: false
    frame_rate: 20
    digital_shift: 0
    bin_x: 0
    bin_y: 0
    roi_x: 0
    roi_y: 0
    roi_width: 1624
    roi_height: 1234
    AcquisitionStartTrigMode: Off
    AcquisitionStartTrigSource: Software
    AcquisitionStartTrigActivation: RisingEdge
    FrameStartTrigMode: On
    FrameStartTrigSource: Software
    FrameStartTrigActivation: RisingEdge
    line_selector: Out1
    line_mode: Output
    line_source: ExposureActive
    user_output_selector: UserOutput1
    info_filename: info_left.ini